.github/workflows/workspace.yml

```diff
@@ -49,6 +49,10 @@ jobs:
       env:
         RUSTFLAGS: -Dwarnings --cfg getrandom_backend="wasm_js"
       run: cargo clippy -Zbuild-std --target wasm32-unknown-unknown
+    - name: Web WASM with atomics (wasm_js.rs)
+      env:
+        RUSTFLAGS: -Dwarnings --cfg getrandom_backend="wasm_js" -Ctarget-feature=+atomics,+bulk-memory
```

Comment:
What's the reason for needing `bulk-memory` here?

Reply:
LLVM requires the bulk memory proposal when using the threads (called "atomics") proposal.
However, I can see that LLVM does this check only on the linker level, so its not required for Clippy.

Let me know if you want me to remove it.

Reply:
Makes sense to keep as-is. We should keep the flags the same between clippy and testing. Thanks for the explanation.